azure-pipelines/build.yml

```diff
@@ -42,7 +47,7 @@ extends:
             publishVstsFeed: 97a41293-2972-4f48-8c0e-05493ae82010
           - output: nuget
             displayName: 'NuGet push to NuGet.org'
-            condition: and(and(succeeded(), eq(variables['PublishNuGetOrg'], 'true')), eq(variables['TimestampPackage'], 'false'))
+            condition: and(succeeded(), eq(variables['Build.OfficialRelease'], 'true'))
             packageParentPath: '$(Build.ArtifactStagingDirectory)\Packages'
             packagesToPush: $(Build.ArtifactStagingDirectory)\Packages\*.nupkg;!$(Build.ArtifactStagingDirectory)\Packages\*.symbols.nupkg
```

Comment:
Let's remove `;!$(Build.ArtifactStagingDirectory)\Packages\*.symbols.nupkg` since this is only a content package